vignettes/transition_disruption_metric.Rmd

```diff
@@ -0,0 +1,552 @@
+---
+title: "Transition Disruption Metric"
+author: "Jackson Hoffart"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{transition_disruption_metric}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(dplyr)
+library(ggplot2)
+library(PACTA.analysis)
+```
+
+# Introduction
+
+PACTA offers many different metrics, each with their own strengths and weaknesses. Most of the metrics rely heavily on the use of forward-looking data. This data, which estimates the production of physical assets, is reliably predictable to about five years into the future. 
+
+The other key component to these metrics is a climate scenario, which projects how production *should* shift, to mediate climate change. There are, however, certain climate scenarios which deliberately offset the bulk of their ambition to the later years in the scenario (sometimes 20+ years into the future). A "delayed response" scenario, such as the  Inevitable Policy Response (IPR) Scenario, assumes that future production of most sectors follow current trends (i.e. business as usual), before inevitably resulting in a massive and disruptive transition in the long-term. Since the interesting part of these scenarios is often positioned well beyond the 5 year time-horizon of the data, most usual PACTA metrics (eg. Volume Trajectory) are ill-suited to provide useful insights. 
+
+The purpose of these "delayed response" scenarios is to estimate the risks associated with a "worst-case scenario". With such a scenario, we might want to measure how **disruptive** a climate **transition** could be to a company or portfolio.  
+
+To help provide insights into this question, we propose the Transition Disruption Metric (TDM). This metric aims to provide useful insights in the medium- to long- term (ie. time-horizons greater than 5 years).
+
+# Derivation
+
+We will design the metric together, in a few steps. Consider some function, $S(t)$, that in general may increase or decrease over time:
+
+```{r, echo = FALSE}
+
+data <- tibble::tribble(
+  ~production, ~year, ~type,
+  1L, 2020L, "S(t)",
+  2L, 2025L, "S(t)",
+  3L, 2030L, "S(t)",
+  4L, 2035L, "S(t)",
+  5L, 2040L, "S(t)",
+  6L, 2045L, "S(t)",
+  7L, 2050L, "S(t)",
+  1L, 2025L, "P(t)"
+)
```

Comment:
You may need to use `# styler: off` and `# stlyer: on` to keep the column names and values aligned.